File: help/eng_gb/exercicios_da_ficha_9/Ficha09_014/Ficha09_014.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha09_014 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Decimal : " );
        int num = Integer.valueOf(keyb.nextLine());
        String binary = "" ;
        while(num > 0 ){
            binary = num % 2 + binary ;
            num /=2;
        }
        System.out.print("Binary = " + binary );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_014/Ficha09_014.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Decimal : " ;
    $num = intval(isset($_POST["num"])? $_POST["num"] : isset($_GET["num"]) ? $_GET["num"] : 0);
    
    echo $num . "<br/>";// print input 
    $binary = "" ;
    while($num > 0 ){
        $binary = $num % 2 . $binary ;
        $num = $num / 2 ;
    }
    echo  "Binary = " . $binary  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Sat, 2022 Oct 01 00:12

?>